display/ssd1306.h

```diff
@@ -445,66 +459,127 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
     }
   }
 
+  void SB_On2() override {
+    if (IMG_out) {
+      float dur = (font_config.ProffieOSOutImageDuration == 0) ? SaberBase::sound_length * 1000 : font_config.ProffieOSOutImageDuration;
```

Comment:
This gets repeated a lot.
Can we make a helper function?

Reply:
100% yes.
I just hadn't worked that out yet. Check now :)